projects/common/src/utilities/formatters/time/display-time-ago.pipe.ts

```diff
@@ -0,0 +1,48 @@
+import { Pipe, PipeTransform } from '@angular/core';
+import { Dictionary } from '../../types/types';
+
+@Pipe({
+  name: 'htDisplayTimeAgo'
```

Comment:
This is not a pure pipe - the results will change over time. How do we want to handle that from a change detection point of view?

Reply:
Yeah I go back and forth here. For the column, I agree with you. When data reloads, we blow away the whole table and redraw anyway. For a generic pipe, that's where I hesitate. For example, if we were to use this in the refresh button (which actually does show "time ago"), it wouldn't update.

tl;dr - I'm ok leaving it pure as long as you comment about this

Reply:
Comment added.